### ISSUE TITLE
v48

### ISSUE THREAD

**Issue by JeanlChristophe** [synonymdev/bitkit-react-native#1864]
Bitkit Mobile Prototype v48 (HANDOFF) (LAST ONE FOR V1.0)
- These changes are ready to be implemented in frontend and backend.
- Not all changes are visible in the prototype flows, please check the static Figma frames as well.
- This version excludes ongoing experimental design work.

- [x]  The Beta to v1.0 migration handholding UI will be available tomorrow.
New user: https://www.figma.com/proto/ltqvnKiejWj0JQiqtDf2JJ/Bitkit-Wallet?page-id=24334%3A157948&node-id=24334-163280&viewport=200%2C202%2C0.13&t=AiaJa0vXi9WBUfOB-8&scaling=min-zoom&starting-point-node-id=24334%3A163280&hide-ui=1
Active user: https://www.figma.com/proto/ltqvnKiejWj0JQiqtDf2JJ/Bitkit-Wallet?page-id=24334%3A157948&node-id=24334-163284&viewport=200%2C202%2C0.13&t=AiaJa0vXi9WBUfOB-8&scaling=min-zoom&starting-point-node-id=24334%3A163284&show-proto-sidebar=1&hide-ui=1

MINOR CHANGES

- [x] Contrast: Changed alpha values for copy from #FFFFFF 50% to #FFFFFF 64%.
- [x] Onboarding: Updated copy for various onboarding frames (headings + paragraphs).
- [x] Onboarding > Wallet: Placed emphasis on ‘Don’t store all your money in Bitkit’.
- [x] Wallet overview: Removed the ‘Don’t store all your money in Bitkit’ warning card from the wallet overview.
- [x] Suggestion cards: Changed letter spacing from -1px to -0.5px for the 20px headings on the cards
- [x] Suggestion cards: Updated default sets for empty and funded wallet states
- [x] Suggestion card ‘SPEND’: Changed second line to ‘Instant payments’ to highlight the benefit of funding and setting up a spending balance.
- [x] Progress card ‘TRANSFER’: Changed second line to only ‘In progress..’ for all steps of the LN setup process. User can tap on card to see more details about the process in the full screen setup flow.
- [x] Progress card ‘FAILED’: Changed colors for this card to Bitkit orange.
- [x] Widgets: Added the Bitcoin price widget (1D USD) as a default preinstalled widget on fresh installs.
- [x] Toasts: Please add 1 second to ‘Switched to dollars’, Wallet Balance Hidden’, and ‘Spending Balance Ready’
- [x] Savings Overview: Added empty state
- [x] Spending Overview: Added empty state
- [x] Buy Bitcoin: Changed heading to: ‘BUY SOME BITCOIN’ and label of button to ‘Choose Exchange’
- [x] Connection Closed: Changed copy to: ‘Your Lightning Connection has closed. Your funds transferred to your savings.’
- [x] Restore error: Changed copy to: ‘SPENDING BALANCE ERROR’
- [x] Restore error: Updated the paragraph copy for the spending balance error.
- [x] Update modal: Updated visual design and layout.
- [x] Forced update: Updated visual design and layout.
- [x] Force transfer modal: Updated visual design and layout.
- [x] Transfer failure modal: Updated visual design and layout.
- [x] Transfer failure modal: Changed graphic to 3D ‘X’ icon.
- [x] High balance modal: Updated visual design and layout.
- [x] Profile onboarding: Changed paragraph copy.
- [x] Contacts onboarding: Changed graphic to 3D ‘users’ icon.
- [x] Widgets onboarding: Removed second onboarding screen (passwordless accounts)
- [x] Buttons: Added pressed state for regular buttons (see design system)
- [x] Backup: Removed ‘Export wallet data to phone’ option
- [x] Settings > General: Added setting for user to choose between orange or black app icon (default=orange)
- [x] ~Max amount: Added max amount modal with short explanation for end user for when ‘max’ amount is tapped in spending balance send flow. https://synonymworkspace.slack.com/archives/C01H106JRQB/p1712855096363569?thread_ts=1712850323.807499&cid=C01H106JRQB~
- [ ] Transaction Failed: Added ‘Send Anonymous Report’ button. https://synonymworkspace.slack.com/archives/C01H106JRQB/p1714565582815419?thread_ts=1714550459.184499&cid=C01H106JRQB
- [x] Backup flow: Rearranged ‘Successful’ and ‘Keep It Secret’, changed title to ‘Keep It Safe’. https://synonymworkspace.slack.com/archives/C01J0QPPV08/p1714997788429509
- [x] Transfer to Savings frames: Updated paragraph copy for channel closure frames. Changed channel closure confirmation graphic to 3D checkmark.
- [x] Settings > Close Connection: Updated paragraph copy for channel closure frame.
- [x] Transfer > In Transfer: Updated paragraph copy (removes the necessity to keep app open).

**Comment by catch-21** [comment]
1. "Changed second line to ‘Instant payments’" this hasn't been done in Figma. It says 'instant and cheap transactions"

2. "Backup flow: Rearranged ‘Successful’ and ‘Keep It Secret’, changed title to ‘Keep It Safe’." hasn't been done. Still "Keep It Safe' followed by 'Successful'

**Comment by JeanlChristophe** [comment]
Can we close this ticket?

**Comment by pwltr** [comment]
No, some tasks (bullet points) are not done yet. They are low prio imo.

**Comment by JeanlChristophe** [comment]
@dzdidi @SeverinAlexB how/where do you would like to receiv Transaction Failed: Added ‘Send Anonymous Report’ button. https://synonymworkspace.slack.com/archives/C01H106JRQB/p1714565582815419?thread_ts=1714550459.184499&cid=C01H106JRQB report?

**Comment by SeverinAlexB** [comment]
Use the log function as mentioned by Jay.
https://synonymworkspace.slack.com/archives/C01H106JRQB/p1714552590905639?thread_ts=1714550459.184499&cid=C01H106JRQB

If you actually want to do something with this data then we need to create a ticket on our side. So far, the logs are only getting posted in the Slack log channel.

**Comment by JeanlChristophe** [comment]
https://github.com/synonymdev/bitkit/issues/2003 related

**Comment by pwltr** [comment]
Moved the 'Max Amount' subtask to https://github.com/synonymdev/bitkit/issues/2003
